packages/backend-defaults/src/entrypoints/urlReader/lib/BitbucketCloudUrlReader.ts

```diff
@@ -26,13 +26,15 @@ import {
 import { NotFoundError, NotModifiedError } from '@backstage/errors';
 import {
   BitbucketCloudIntegration,
+  BitbucketFetchFunction,
+  BitbucketFetchService,
```

Comment:
I really would want to get rid of these types. The core of it is, the integrations package should be left as pristine as is humanly possible. See comments below.